requirements.in

```diff
@@ -0,0 +1,3 @@
+httpx
+jinja2
```

Comment:
jinja2 is a dependency of render_engine. You don't really need to import it (Even though we're using it to compile the templates)

Reply:
It's also a top-level dependency, though. If the dependencies of `render_engine` ever changed to drop `jinja2`, `pip-compile` would remove it unless explicitly listed (or unless it was a dependency of another dependency). The TL;DR is that `requirements.in` should generally contain anything you use directly, and `requirements.txt` gets compiled to include a full set of dependencies all the way down.

Reply:
makes sense to me. Approved and ready to merge